Sprint-3/revise/implement/repeat.js

```diff
@@ -0,0 +1,18 @@
+
+
+function repeat(str, count) {
+    
+    if (count < 0) {
+        throw new Error("Count cannot be negative");
+    }
+
+    
+    if (count === 0) {
+        return "";
+    }
+
+    
+    return str.repeat(count);
```

Comment:
This is good but it would be better if you could implement the repeat functionality

Reply:
Hi, Thanks for the review. Resolved